unit-tests/src/testing_utils/neutron_adapters/stargate_module.rs

```diff
@@ -0,0 +1,126 @@
+use cosmos_sdk_proto::traits::MessageExt;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{
+    coins, from_json, to_json_binary, Addr, AnyMsg, Api, Binary, BlockInfo, Coin, CustomMsg,
+    CustomQuery, GrpcQuery, Querier, Storage, Uint64,
+};
+use cw_multi_test::{
```

Comment:
Doesn't multi-test have stargate implementation? pretty sure i saw something like that.
Just checked, they don't.